src/App.css

```diff
@@ -40,7 +40,13 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  color: white;
+  font-size: 72px;
+   background: -webkit-linear-gradient(left, rgb(255, 0, 0) , rgb(25, 0, 248));
+   background: -o-linear-gradient(right, rgb(255, 0, 0), rgb(25, 0, 248));
+   background: -moz-linear-gradient(right, rgb(255, 0, 0), rgb(25, 0, 248));
+   background: linear-gradient(to right, rgb(255, 0, 0) , rgb(25, 0, 248)); 
```

Comment:
First question, why multiple background properties? 
Another question: why setting font-size twice? you added at line 43, and there's one already on 50
I think one of the font-size needs to new removed(probably your's because the other one is dynamic)

Reply:
My bad, I will try to fix it asap.

Reply:
![Screenshot (172)](https://user-images.githubusercontent.com/92904175/195330493-7337cdcc-8d53-4e32-9d1f-afbe67f84fca.png)

I have resolved the issues and made the gradient more pretty and clear. Please do check.